pkg/cluster/fetch_test.go

```diff
@@ -2,43 +2,48 @@ package cluster_test
 
 import (
 	"context"
+	"errors"
 	"testing"
 
+	eksdv1 "github.com/aws/eks-distro-build-tooling/release/api/v1alpha1"
+	"github.com/golang/mock/gomock"
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
```

Comment:
I did a bunch of package renames to follow our latest convention and because it was getting confusing with the old names